src/briefcase/integrations/linuxdeploy.py

```diff
@@ -119,3 +119,71 @@ def patch_elf_header(self):
                 # The original or patched value. If this is the case, the file is likely
                 # wrong and we should raise an exception.
                 raise CorruptToolError("linuxdeploy")
+
+
+class LinuxDeployGtkPlugin:
+    name = "linuxdeploy_gtk_plugin"
+    full_name = "linuxdeploy_gtk_plugin"
+
+    def __init__(self, command):
+        self.command = command
+
+    @property
+    def plugin_name(self):
+        return "linuxdeploy-plugin-gtk.sh"
```

Comment:
What's the functional difference between `name`, `full_name`, and `plugin_name`?

Reply:
You are right, I don't think they are needed, I removed everything but the `plugin_name`. :+1: